SETUP/upgrade/13/20190929_update_user_project_info.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+$relPath='../../../pinc/';
+include_once($relPath.'base.inc');
+include($relPath.'udb_user.php');
```

Comment:
```suggestion
include($relPath.'udb_user.php'); // $archive_db_name
```

(I wondered why the script would need that include.)